### DISCUSSION TITLE
Inquiry regarding accessing eqations used in FABM

### DISCUSSION THREAD

**Discussion by Julia-Cheng0225** [fabm-model/fabm#40] [Q&A] [answered]
Hi,
I am a new user of fabm and am trying to use the fabm-gotm-npzd testcase by pyfabm. However, I have been unable to find detailed information about the equations describing the biochemistry process.
If possible, could you please tell me where I can find information about the equations used in npzd models within pyfabm?
Thank you!

**Comment by jornbr** [comment, marked as answer]
Hi, the underlying code can be found at [`src/models/gotm/npzd.F90`](https://github.com/fabm-model/fabm/blob/master/src/models/gotm/npzd.F90). The header of this file shows the underlying equations (in LaTeX format) and also includes references to the papers where the model is described.

**Comment by Julia-Cheng0225** [comment]
Thank you so much.
